docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -0,0 +1,240 @@
+                  <div className="key">nfts_uploaded</div>
+                  <div>{mintlist.mint_infos.length}</div>
+                  <div className="key">nfts_total</div>
+                  <div>{mintlist.num_nfts_total}</div>
+                  <div className="key">nfts_minted</div>
+                  <div>{mintlist.num_nfts_redeemed}</div>
+                </div>
+              </div>
+            </div>
+          </>
+        )}
+      </PageLayout>
+      <style jsx>{`
+        .navigation {
+          margin-bottom: 2rem;
+        }
+
+        .columns {
+          display: grid;
+          grid-template-columns: 20rem 1fr;
+          grid-column-gap: 3rem;
+        }
+
+        .table {
+          padding: 0.75rem;
+          border-radius: var(--border-radius);
+          background-color: var(--secondary-bg-color);
+          font-family: var(--mono-font);
+          font-weight: var(--medium-font-weight);
+          overflow-wrap: anywhere;
+
+          display: grid;
+          grid-template-columns: max-content 1fr;
+          grid-column-gap: 2rem;
+        }
+
+        .table .key {
+          font-weight: var(--normal-font-weight);
+          color: var(--secondary-color);
+          max-width: 8.5rem; /* This width cuts "authority_can_update" in a nice way */
+        }
+
+        img {
+          display: block;
+          width: 100%;
+          box-shadow: var(--shadow);
+          border-radius: calc(var(--border-radius) * 2);
+        }
+
+        figcaption {
+          margin-top: 1rem;
+          text-align: center;
+          font-weight: bold;
+        }
+
+        @media (max-width: ${ResponsiveBreakpoint.medium}) {
+          img {
+            max-width: 24rem;
+            margin: 0 auto;
+          }
+
+          h1,
+          h2 {
+            text-align: center;
+            margin-bottom: 1.5rem;
+          }
+
+          .columns {
+            grid-template-columns: 1fr;
+            grid-row-gap: 1.5rem;
+          }
+
+          .collection {
+            // Push the collection section to the bottom.
+            order: 1;
+          }
+        }
+      `}</style>
+    </>
+  );
+}
+
+type MintlistInfoWithCollectionInfo = NftokenTypes.MintlistInfo & {
+  collectionInfo: NftokenTypes.CollectionInfo | null;
+};
+
+function useMintlist({
+  address,
+  network,
+}: {
+  address: Solana.Address;
+  network: Network;
+}): {
+  data?: MintlistInfoWithCollectionInfo | null;
+  error: any;
+  mutate: SWRResponse<MintlistInfoWithCollectionInfo | null, never>["mutate"];
+} {
+  const swrKey = [address, network];
+  const { data, error, mutate } = useSWR(swrKey, async () => {
+    const mintlist = await NftokenFetcher.getMintlist({ address, network });
+
+    if (!mintlist) {
+      return null;
+    }
+
+    const collection = await NftokenFetcher.getCollection({
+      address: mintlist.collection,
+      network,
+    });
+
+    return {
+      ...mintlist,
+      collectionInfo: collection,
```

Comment:
Hmm, a nicer type could be: `{mintlist, collection}`. That's a bit more clear / expected.